src/NearOrg/Notifications/SettingsListItem.jsx

```diff
@@ -68,15 +105,17 @@ return (
         </Text>
       </Wrapper>
     </Content>
-    <Widget
-      src="${REPL_ACCOUNT}/widget/DIG.Button"
-      props={{
-        label,
-        variant: "primary",
-        onClick: handleOnClick,
-        loading,
-        disabled,
-      }}
-    />
+    <SwitchWrapper>
+      <Switch.Root
+        data-state={isSwitchChecked ? "checked" : "unchecked"}
```

Comment:
I think it's better to use `checked` prop ([doc](https://www.radix-ui.com/primitives/docs/components/switch#root)) here instead of this conditions. And no need to use `data-state` for `Switch.Thumb` component

Reply:
After I close this one, I will create a separate PR for this suggestion. Thank you.